content/docs/api-reference/artifacts_show.md

```diff
@@ -0,0 +1,102 @@
+# dvc.api.artifacts_show()
+
+Get the path and Git revision for an artifact tracked in a <abbr>DVC
+repository</abbr>.
+
+```py
+def artifacts_show(
+    name: str,
+    version: Optional[str] = None,
+    stage: Optional[str] = None,
+    repo: Optional[str] = None,
+) -> Dict[str, str]:
+```
+
+## Usage:
+
+```py:
+import dvc.api
+
+artifact = dvc.api.artifacts_show(
+    'text-classification',
+    repo='https://github.com/iterative/example-get-started.git',
+)
+```
+
+## Description
+
+Returns a path and Git revision for a named artifact which can then be used in
+other Python API calls.
+
+The returned dictionary will be of the form:
+
+```py
+{
+    'path': 'model.pkl',
+    'rev': 'c7c6ae0',
+}
+```
+
+where `path` contains the relative path to the artifact in the DVC repository,
+and `rev` contains the Git revision for the specified artifact version or stage.
+
+When neither `version` nor `stage` are provided, the Git revision for the latest
+version of the model will be returned.
+
+## Parameters
+
+- `name` (required) - name of the artifact.
```

Comment:
We should at least mention how to address artifacts stored outside the root `dvc.yaml`.